### PR TITLE
2108/pinode in policy

### PR DESCRIPTION
**Pull request by cornelinux**

### CODE REVIEW

File: privacyidea/api/policy.py [hunk 1]
```diff
@@ -544,6 +546,8 @@ def get_policy_defs(scope=None):
             "sections": section_descriptions,
             "comparators": comparator_descriptions,
         }
+    elif scope == 'pinodes':
+        result = get_privacyidea_nodes()
```
[the remark this answers is not in the document]
Comment (cornelinux) [reply privacyidea/api/policy.py:550]: Will check and do.

File: privacyidea/config.py [hunk 2]
```diff
@@ -115,6 +115,8 @@ class ProductionConfig(Config):
     PI_AUDIT_KEY_PUBLIC = os.path.join(basedir, "public.pem")
     PI_LOGLEVEL = logging.INFO
     SUPERUSER_REALM = ['superuser']
+    PI_NODE = "Node1"
+    PI_NODES = ["Node1", "Node2"]
```
[the remark this answers is not in the document]
Comment (cornelinux) [reply privacyidea/config.py:119]: No. This is basically only for development and testing. 
After all, a *real* installation should have set its notes in pi.cfg manually anyways.

[the remark this answers is not in the document]
Comment (plettich) [reply privacyidea/config.py:119]: exactly, an external `pi.cfg` only overwrites the parameter in the production config.

File: privacyidea/api/policy.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (cornelinux) [reply privacyidea/api/policy.py:552]: Done in #3e14250b

File: privacyidea/config.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (cornelinux) [reply privacyidea/config.py]: OK, honestly I added these to the production config for manual tests. But it is a bad idea, so I remove these in 3e14250b. If the production config or pi.cfg does not contain any further, specifc configuration we end up with "localnode" in the UI, which is imho a very sensible and correct way for a single node installation.
In all cases with multiple nodes, admins need to configure the nodes in pi.cfg anyways.

**Pull request merged**